libs/ballot-interpreter/src/hmpb-rust/layout.rs

```diff
@@ -33,11 +33,9 @@ fn build_option_layout(
     let column_offset = -(grid_layout.option_bounds_from_target_mark.left as i32);
     let row_offset = -(grid_layout.option_bounds_from_target_mark.top as i32);
     let width: GridUnit = grid_layout.option_bounds_from_target_mark.left
-        + grid_layout.option_bounds_from_target_mark.right
-        + 1;
+        + grid_layout.option_bounds_from_target_mark.right;
     let height: GridUnit = grid_layout.option_bounds_from_target_mark.top
-        + grid_layout.option_bounds_from_target_mark.bottom
-        + 1;
+        + grid_layout.option_bounds_from_target_mark.bottom;
```

Comment:
Here's the off-by-one issue. We're adding in 1 to the width and the height, in addition to the `top + bottom` or `left + right` specified by the `gridLayouts`. I imagine the original thought here was to account for the size of the bubble itself? But we don't need to do that, the measurements are from the center of the bubble in all directions.

Reply:
Weird, I don't know why we would have done that! Nice catch